src/main/java/seedu/duke/commands/MarkCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.duke.commands;
+
+import seedu.duke.data.Item;
+import seedu.duke.data.ShoppingList;
+
+public class MarkCommand extends Command {
+    public static final String COMMAND_WORD = "MARK";
+    public static final String SUCCESS_DONE = "Yes! I've marked this item as bought: "
+                                                + System.lineSeparator() + "%s";
+    public static final String FAIL_DONE = "Oh No! This item does not exist in the list";
```

Comment:
this could be MESSAGE_FAILURE/FAILURE_MESSAGE (again.. im not screaming at you haha 🤣)

Reply:
Thanks!! Will change it